yarn.lock

```diff
@@ -0,0 +1,1462 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
It is necessary to add a `yarn.lock` when there is already a `package-lock.json` ?